packages/cli/src/utils.ts

```diff
@@ -142,6 +142,7 @@ export function showDeployment (deploymentResult: GSNContractsDeployment, title:
   Penalizer: ${deploymentResult.penalizerAddress}
   VersionRegistry: ${deploymentResult.versionRegistryAddress}
   Forwarder: ${deploymentResult.forwarderAddress}
+  TestToken (test only): ${deploymentResult.testTokenAddress}
```

Comment:
if it is "test only", then don't dump it always.

Reply:
This command will be used 1000 times during testing and once during actual deployment. `AcceptEverythingPaymaster` printed here is also test-only, as a matter of fact.

Reply:
this is the tool that prints during `gsn-deploy`. it shouldn't print test-only stuff 
I don't expect anyone to use it to deploy production networks, but people may use it to deploy on test networks and thus deploy an accept-everything paymaster.

Reply:
I believe this is a helpful string and I will not waste my time removing a line that just stays empty if test token is not deployed. Thank you.